### ISSUE TITLE
[BUG] Problem with ngrok

### ISSUE THREAD

**Issue by Luxid434** [thecoder-001/MineColab#8]
Hi ! I have some problems to connect to the server with the **ip adress provided by ngrok.**
The minecraft client says : Disconnected.

Any thoughts?

Thanks.

**Comment by thecoder-001** [comment]
Hello!
I'm aware of this problem and another person  (#6 ) is too experienced it. It looks like ngrok has imposed some sort of a new limit from a few days ago which doesn't seems to be documented yet and keeps dropping the connection (even plain pinging doesn't work). For now, you can get around this limit by creating another ngrok account and keep cycling between both of them so that limit (however small it is) is not reached.
For a long term solution, I'm thinking of introducing ngrok alternatives which offer higher limits than the currently restricted free plan of ngrok.

Cheers.

**Comment by Luxid434** [comment]
Hi! Sorry i've been not clear : ngrok let me open tunnels properly, and I have the ngrok tunnel's ip :
![Screenshot_20201202-082241_Chrome](https://user-images.githubusercontent.com/66694178/100841448-9a55d100-3477-11eb-9170-2835ab42d417.png)

The problem is that when I want to connect to the server in minecraft, it says disconnected. And I think "disconnected" means that the server is responding, so the tunnel work :
![Screenshot_20201202-081915_Chrome](https://user-images.githubusercontent.com/66694178/100841124-23203d00-3477-11eb-9e2a-87cf3e21feaf.png)
The server is also working properly : 
![Screenshot_2](https://user-images.githubusercontent.com/66694178/100841128-24516a00-3477-11eb-82b1-25db6dce77ec.jpg)
Did you already get into the minecraft server ? 

Thanks for the great work.

**Comment by thecoder-001** [comment]
No problem for less info. Actually, I was just testing some things yesterday using one of my heavily used ngrok account and the client just kept disconnecting. The console had provided me the ip and which was also being shown in ngrok's dashboard, same as yours. I tried pinging the server (to which the server should respond if we are really able to connect to it), but there was no response. The server had started normally and everything looked all right at that part. At least, if "disconnected" was due to server's will, there must have been some log.
However, on trying to use a different fresh ngrok account, it worked without any problems. After multiple tests, it was clearly evident that ngrok was actually allowing to create tunnels but denying any traffic through them for accounts who were used pretty heavily. There is nothing mentioned about a cap on traffic on ngrok's website and people suddenly started to have this issue, it looks to be a recent change. 
So, for now the only solution remains is to create multiple ngrok accounts (and just store all their keys in a text file for easier access) and keep cycling between them so that we don't hit the hidden cap.

Cheers.
